go/auth/credential_authority_test.go

```diff
@@ -44,18 +44,19 @@ func genKID() keybase1.KID {
 	return keybase1.KIDFromSlice(kid[:])
 }
 
+var userID int
+
 func genUsername() string {
-	w, _ := libkb.SecWordList(1)
-	var buf [4]byte
-	rand.Read(buf[:])
-	return fmt.Sprintf("%s%x", w[0], buf)
+	name := fmt.Sprintf("user%d", userID)
+	userID++
```

Comment:
I feel like random is better than `user0`, `user1`, etc. for repeated test runs.  Can you just use `buf [8]byte; rand.Read(buf[:])` or something if you don't want to use libkb.SecWordList?